.claude/skills/release-notes/SKILL.md

```
@@ -0,0 +1,103 @@
+バージョン $ARGUMENTS のリリースノート（PR要約 + ユーザー向けリリースノート）を作成する。
+
+## 出力先
+
+`docs/release_notes/v$ARGUMENTS/`
+
+## 手順
+
+### Step 1: 準備
+
+1. 出力ディレクトリを作成:
+   ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)